validation.js

```diff
@@ -0,0 +1,39 @@
+const form = document.querySelector('.form');
+const email = form.elements['user-email'];
+const message = document.createElement('small');
+message.classList.add('form-message');
+
+form.appendChild(message);
+
+function validateEmail() {
+  const regex = /^[a-z0-9@.]+$/;
+
+  if (email.value.trim() === '') {
+    message.textContent = 'Please enter a valid email-address';
+    email.classList.add('error');
+    console.log('fallse');
+    return false;
```

Comment:
- I tried checking for the validation by entering an email with capital letters, but the form refuse to submit, which is intended, but it should also include a warning message close to the button showing the user to enter email in small letters. Please try fixing that.
- Please the form also refused to submit when I entered the correct format of email, please try fixing it so it submits when the correct inputs are provided.
-  [Optional] I will suggest your use a CSS property of `cursor: pointer;` on hover interaction, on your buttons so it looks more professional when hovered on.

#### Your design and test

https://user-images.githubusercontent.com/58771507/175575233-b3ba52b7-8979-4b3e-80ff-85b78c36605f.mp4